flink-libraries/flink-gelly/src/main/java/org/apache/flink/graph/gsa/GatherSumApplyIteration.java

```diff
@@ -289,6 +300,11 @@ private GatherUdf(GatherFunction<VV, EV, M> gatherFunction, TypeInformation<Tupl
 
 		@Override
 		public void open(Configuration parameters) throws Exception {
+			try {
+				Collection<LongValue> numberOfVertices = getRuntimeContext().getBroadcastVariable("number of vertices");
+				this.gatherFunction.setNumberOfVertices(numberOfVertices.iterator().next().getValue());
+			} catch (Exception e) {
+			}
```

Comment:
BTW: In the IntelliJ code style, they encourage to call ignored exceptions "ignored", or to at least add a comment why the exception is ignored:
```java
try {
    ...
} catch (Exception ignored) {}
```
or
```java
try {
    ...
} catch (Exception e) {
    // thrown if BV does not exist, ignore here
```